src/ui/components/LoadingIndicator/index.js

```diff
@@ -0,0 +1,46 @@
+import classNames from 'classnames';
+import React, { PropTypes } from 'react';
+
+import Icon from 'ui/components/Icon';
+
+import './LoadingIndicator.scss';
+
+
+export default class LoadingIndicator extends React.Component {
+  static propTypes = {
+    altText: PropTypes.node,
+    timeToReset: PropTypes.number,
+  }
+
+  static defaultProps = {
+    altText: null,
+    timeToReset: 500,
+  }
+
+  constructor(props) {
+    super(props);
+    this.state = { animating: true };
+  }
+
+  restartAnimation = () => {
```

Comment:
You shouldn't need any JS to do this. The timing will be slightly different but roughly accurate to what you had.

```scss
.LoadingIndicator {
  animation: spin 1.6s infinite cubic-bezier(0.42, 0.26, 0.04, 1.21);
  height: 48px;
  width: 48px;
}

@keyframes spin {
  0% {
    transform: rotate(0deg);
  }

  66% {
    transform: rotate(720deg);
  }

  100% {
    transform: rotate(720deg);
  }
}